.changeset/red-pots-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+'svelte': patch
+---
+
+fix: style shorthand not referencing variables
```

Comment:
the convention we _try_ to adhere to (with less than 100% success rate) is to describe what happens as a result of the changeset, rather than the issue the changeset addresses

```suggestion
fix: mark variables in shorthand style directives as referenced
```